Dockerfile

```diff
@@ -1,20 +1,16 @@
-FROM debian:jessie
+FROM alpine:latest
 
-# install curl and jq
-RUN apt-get update && \
-    apt-get install -y \
-    curl \
-    jq && \
-    rm -rf /var/lib/apt/lists/*
+ENV CB_RELEASE=1.0.0
+ENV CB=containerbuddy-1.0.0.tar.gz
+WORKDIR /opt/containerbuddy
+
+# install jq
+RUN apk update ; apk add jq ; rm -rf /var/cache/apk/*
 
 # get containerbuddy release
-RUN export CB=containerbuddy-0.0.1-alpha &&\
-    mkdir -p /opt/containerbuddy && \
-    curl -Lo /tmp/${CB}.tar.gz \
-    https://github.com/joyent/containerbuddy/releases/download/0.0.1-alpha/${CB}.tar.gz && \
-	tar -xf /tmp/${CB}.tar.gz && \
-    mv /build/containerbuddy /opt/containerbuddy/
+ADD https://github.com/joyent/containerbuddy/releases/download/$CB_RELEASE/$CB .
+RUN gunzip $CB ; tar xf containerbuddy-1.0.0.tar ; rm containerbuddy-1.0.0.tar
```

Comment:
Because we need `curl` in the image anyways, this doesn't get us anything and adds an extra layer + associated metadata to the build (2 instructions vs 1 instruction).